tools/stats/upload_stats_lib.py

```diff
@@ -143,6 +143,23 @@ def upload_to_s3(
     print("Done!")
 
 
+def read_from_s3(
+    bucket_name: str,
+    key: str,
+) -> List[Dict[str, Any]]:
+    print(f"Reading from s3://{bucket_name}/{key}")
+    body = (
+        S3_RESOURCE.Object(
+            f"{bucket_name}",
+            f"{key}",
+        )
+        .get()["Body"]
+        .read()
+    )
+    results = gzip.decompress(body).decode().split("\n")
```

Comment:
Curious why you would need `split("\n")` here as I assume that decompressing the gzip will return the exact original JSON